src/extensions/extension-loader.ts

```diff
@@ -19,6 +19,9 @@ export class ExtensionLoader {
   protected extensions = observable.map<LensExtensionId, InstalledExtension>();
   protected instances = observable.map<LensExtensionId, LensExtension>();
 
+  /**
+   * @observable
```

Comment:
What does this `@jsdoc`?

Reply:
I don't understand your question. I added these comments so that these variable's observable status would be added to the docs.

Reply:
Alright, it's not clear where this code / comment actually participate :)

Reply:
Would you prefer something more along the lines of `This field is mobx observable`?

Reply:
Since PR should be targeted on docs only? Otherwise why only in some `.ts` files here I see these comments?

Reply:
Because comments are part of documentation